serverless.yml

```diff
@@ -49,7 +49,7 @@ custom:
     SETT_HISTORIC_DATA: sett-historic-snapshots-${self:custom.stage}
     # deprecated
     VAULT_SNAPSHOTS_DATA: vaults-snapshots-${self:custom.stage}
-    VAULT_COMPOUND_DATA: vaults-compound-${self:custom.stage}
+    VAULT_DEFINITION_DATA: vaults-definitions-${self:custom.stage}
```

Comment:
do we have to update the names in serverless_frontend.yml as well?

Reply:
no that is for something else, until that account is ready and provisioned we don't need to worry about it

Reply:
we will fully capture any changes in this serverless there at that point